.github/workflows/release-workflow.yml

```diff
@@ -0,0 +1,45 @@
+name: E4E Release & Publish Workflow
+on:
+  pull_request:
```

Comment:
If we ex. push into dev branch, the pipeline will have run when the PR was created, but not on the `push` action. Is it ok with you? If so I will trigger pipeline only on `master` branch for `push` actions.

Reply:
Yeah, I think it will be perfect.

Because at least in my understanding, the `release` pipeline will be triggered on any `push` event into the master branch, even after a merged PR.